conan/internal/cache/db/cache_database.py

```diff
@@ -90,9 +90,10 @@ def create_recipe(self, path, ref: RecipeReference):
     def create_package(self, path, ref: PkgReference, build_id):
         self._packages.create(path, ref, build_id=build_id)
 
-    def list_references(self):
+    def list_references(self, pattern=None):
```

Comment:
Just add some docstring that explains that this returns the bare ``ref`` without timestamp, etc.